src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -7132,38 +6358,40 @@ namespace VariableSpeedCoils {
         Real64 PLF;                 // part-load function
         Real64 rhoair(0.0);         // entering air density
 
+        auto &varSpeedCoil = state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum);
+
+        if (varSpeedCoil.availSched->getCurrentVal() <= 0.0) {
+            varSpeedCoil.SimFlag = false;
+            return;
+        }
```

Comment:
If the coil is scheduled off, set the `SimFlag` to `false` and return before doing any of the calc.